src/components/details/tests/Details.test.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import Details from '../Details';
+
+describe('Details component', () => {
+  afterEach(() => cleanup());
+  it('renders Details', () => {
+    const { asFragment } = render(<Details />);
```

Comment:
you have the dog details.json that you could import in and spread for details competent. This is will get rid of your prop console error in jest and github.